go.mod

```diff
@@ -527,6 +526,4 @@ replace (
 	github.com/golang/glog => github.com/elastic/glog v1.0.1-0.20210831205241-7d8b5c89dfc4
 	github.com/google/gopacket => github.com/elastic/gopacket v1.1.20-0.20211202005954-d412fca7f83a
 	github.com/lestrrat-go/jwx => github.com/lestrrat-go/jwx v1.2.27
-	go.opentelemetry.io/otel/metric v0.38.0 => go.opentelemetry.io/otel/metric v0.34.0
-	oras.land/oras-go => oras.land/oras-go v1.2.4-0.20230801060855-932dd06d38af
```

Comment:
any reason why these don't need to be replaced anymore?

Reply:
Yes, we specifically need `oras.land/oras-go@v1.2.5` 

buildkit 0.12.5 needs docker v25
https://github.com/moby/buildkit/blob/v0.12.5/go.mod#L31


But oras-go 1.2.4 breaks with newer docker so we need `1.2.5` (check `Important` in the release notes)

https://github.com/oras-project/oras-go/releases/tag/v1.2.5

---

the other one `go.opentelemetry.io/otel/metric v0.38.0 => go.opentelemetry.io/otel/metric v0.34.0` is not useful anymore since we have metric `v1.24.0` so the replace does nothing to sum.